addons/mail/models/mail_channel.py

```diff
@@ -579,7 +763,8 @@ def channel_create(self, name, privacy='public'):
             'name': name,
             'public': privacy,
             'email_send': False,
-            'channel_partner_ids': [(4, self.env.user.partner_id.id)]
+            'channel_partner_ids': [(4, self.env.user.partner_id.id)],
+            'channel_last_seen_partner_ids': [(0, 0, {'partner_id': self.env.user.partner_id.id})]
```

Comment:
Isn't it unnecessary as channel_partner_ids should already create an entry in the m2m link table ?

Reply:
I have erased the new line.